includes/definitions/discovery/algcom-dc-ups.yaml

```diff
@@ -0,0 +1,172 @@
+mib: ALGPOWER-v2-MIB
+modules:
+    os:
+        sysDescr_regex: '/ALGCOM/'
```

Comment:
There are no capture groups in this regex, so it won't do anything.

Reply:
Indeed, i'll clean that up shortly